ifrs17/Test/ReportStorageTest.ipynb

```diff
@@ -119,24 +120,6 @@
             "execution_count": 0,
             "outputs": []
         },
-        {
-            "cell_type": "code",
-            "source": [
-                "activity.Status.Should().Be(ActivityLogStatus.Succeeded);"
```

Comment:
revert

Reply:
@dcolleoni I am not sure what I should revert here...

Reply:
reverted